app/src/main/java/org/oppia/app/customview/SegmentedCircularProgressView.kt

```diff
@@ -79,11 +87,13 @@ class SegmentedCircularProgressView : View {
       val endRight = 2 * radius - startTop
 
       // baseRect will define the drawing space for drawArc().
-      baseRect = RectF(startLeft.toFloat(), startTop.toFloat(), endRight.toFloat(), endBottom.toFloat())
+      baseRect =
+        RectF(startLeft.toFloat(), startTop.toFloat(), endRight.toFloat(), endBottom.toFloat())
     }
 
     var angleStartPoint = -90f
 
+    /* ktlint-disable max-line-length */
```

Comment:
Why are we disabling this? We can break this down right?

Reply:
ok